src/mjolnir/valhalla_add_elevation.cc

```diff
@@ -30,95 +32,61 @@ enum Input { CONFIG, TILES };
  * - tiles - stays for the path to a tile file.
  * */
 
-std::pair<std::string, std::vector<std::string>> parse_arguments(int argc, char** argv) {
-  opt::Options options(
-      "help",
-      " Usage: valhalla_add_elevation [options]\n"
-      "valhalla_add_elevation is a tool for loading elevations for a provided tile. "
-      "The service checks if required elevations stored locally if they are not "
-      "it tries to establish connection to the remote storage(based on the information from configuration file)"
-      "and loads required elevations.\n");
-
-  options.add_options()("h,help",
-                        "Print usage")("c,config", "Path to the configuration file.",
-                                       opt::value<
-                                           std::string>())("t,tiles", "Tiles to add elevations to",
-                                                           opt::value<std::vector<std::string>>());
-
-  options.allow_unrecognised_options();
-  std::string config_file;
+int main(int argc, char** argv) {
+  const std::string program = "valhalla_add_elevation";
```

Comment:
you can get this by messing around with `__FILE__` then you dont have to type every single one though i guess it has to be the name of the source file (which we do adhere to)

Reply:
jep, that worked, thanks! made it a function in `argparse_utils.h`